package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@uniswap/sdk-core",
   "license": "MIT",
-  "version": "4.0.3",
+  "version": "4.0.5",
```

Comment:
note: 4.0.5 is the version _without_ base. i'll be publishing the next version, with base re-added